tree/dataframe/test/datasource_ntuple.cxx

```diff
@@ -132,6 +134,21 @@ TEST_F(RNTupleDSTest, CardinalityColumn)
    EXPECT_EQ(3, *max_rvec2);
 }
 
+// TODO(jblomer): this test will change once collections are read as RVecs in RNTupleDS
+TEST_F(RNTupleDSTest, ReadRVec)
+{
+   auto df = ROOT::RDF::Experimental::FromRNTuple(fNtplName, fFileName);
+
+   // Allow use of float and Float_t interchangibly
+   EXPECT_DOUBLE_EQ(3.0, *df.Sum<std::vector<Float_t>>("jets"));
+   // Allow use of std int types and ROOT int types interchangibly
+   EXPECT_EQ(1U, df.Take<std::uint32_t>("nevent").GetValue()[0]);
+   EXPECT_EQ(1U, df.Take<UInt_t>("nevent").GetValue()[0]);
+   // jets is currently exposed as std::vector<float> and thus not usable as ROOT::RVec<float>
+   EXPECT_ANY_THROW(df.Sum<ROOT::RVec<float>>("jets"));
+   // EXPECT_THROW(df.Sum<ROOT::RVec<float>>("jets"), std::runtime_error); // This does not work directly, maybe due to jitting ?
```

Comment:
We should understand this before merging.

Reply:
Yep, the test is still failing even after that change, so the attempt was not useful